ccs-scale-infra-shared/terraform/modules/cloudtrail/main.tf

```diff
@@ -0,0 +1,174 @@
+#########################################################
+# CloudTrail
+#
+# Cloud Trail and CloudWatch Alarms
+#########################################################
+provider "aws" {
```

Comment:
Was there a reason for re-declaring the provider / switch role here?

Reply:
@tommyb82 - I thought it was needed initially - as the current provider is set in the config, and this sidestepped that (originally). However - I wondered if it really was needed if this was called after that code - but I couldn't get it to work - so I have refactored - see comment below